linter/allowlists/en.txt

```diff
@@ -203,3 +203,9 @@ Watchlist
 watchlist
 yes
 youtube
+auto
+deployment
+release
+Releases
+stages
+Stage
```

Comment:
Some of these are regular English words, so they do not need to be on the English allow list.

Reply:
So i have to commit these words to de.txt? Because some of the words are used in de like "Release" or "Stage"

Reply:
Yes, `Releases` and `Stage` need to be added to the `de` allow list:

```
Error: ] [Package: brockhaus-ag/contao-release-stages-bundle; Language: de]:    
         Property "description" does not pass the spell checker. Either update  
         the allow list or fix the spelling :) Errors: Releases, Stage
```